api/client/application/client_test.go

```diff
@@ -1069,40 +1110,23 @@ func (s *applicationSuite) TestScaleApplicationValidation(c *gc.C) {
 }
 
 func (s *applicationSuite) TestScaleApplicationError(c *gc.C) {
-	apiCaller := basetesting.APICallerFunc(
-		func(objType string, version int, id, request string, a, response interface{}) error {
-			c.Assert(request, gc.Equals, "ScaleApplications")
-			args, ok := a.(params.ScaleApplicationsParams)
-			c.Assert(ok, jc.IsTrue)
-			c.Assert(args, jc.DeepEquals, params.ScaleApplicationsParams{
-				Applications: []params.ScaleApplicationParams{
-					{ApplicationTag: "application-foo", Scale: 5},
-				}})
-
-			result, ok := response.(*params.ScaleApplicationResults)
-			c.Assert(ok, jc.IsTrue)
-			result.Results = []params.ScaleApplicationResult{
-				{Error: &params.Error{Message: "boom"}},
-			}
-			return nil
-		},
-	)
-	client := application.NewClient(apiCaller)
-	_, err := client.ScaleApplication(application.ScaleApplicationParams{
-		ApplicationName: "foo",
-		Scale:           5,
-	})
-	c.Assert(err, gc.ErrorMatches, "boom")
-}
+	ctrl := gomock.NewController(c)
+	defer ctrl.Finish()
 
-func (s *applicationSuite) TestScaleApplicationCallError(c *gc.C) {
```

Comment:
Why was TestScaleApplicationCallError removed? Even mocked, it's different from TestScaleApplicationError.